OutOfSchool/OutOfSchool.Redis/IReadWriteCacheService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.Redis;
+
+public interface IReadWriteCacheService
+{
+    Task<string> ReadAsync(string key);
+
+    public Task WriteAsync(
```

Comment:
(Code style) Please remove “public” as it is public anyway

Reply:
Ok, I've done it